content/en/integrations/_index.md

```diff
@@ -83,10 +85,6 @@ cascade:
     path: /integrations/komodor_license.md
   aliases:
     - /integrations/komodor_komodor/
-- _target:
-    path: /integrations/shoreline_license.md
-  aliases:
```

Comment:
Does the alias here need to be added in the list above?

Reply:
Good point, it probably does.